src/services/temp.java

```diff
@@ -1,5 +1,15 @@
 package services;
 
-public class temp {
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class temp
```

Comment:
Delete temp.java and create your own services class for this service. Update you local branch, recommit, push to update this pull request